README.md

```diff
@@ -94,10 +94,12 @@ pip install crewai[tools]
 
 ### Development Setup
 
+- Create virtual env for Python 3.11: `uv venv --python 3.11`
```

Comment:
required in the pre-commit hooks:
https://github.com/crewAIInc/crewAI-tools/blob/main/.pre-commit-config.yaml#L7

Reply:
we'd recommend using 3.12, we support >3.10, <=3.12

Reply:
@lucasgomide see the linked line of code in the comment above. I can change this to 3.12 if you guys are fine with me updating the black config as well, but this might result in some linting issues